libs/gltfio/include/gltfio/SimpleViewer.h

```diff
@@ -0,0 +1,322 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by mIcable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef GLTFIO_SIMPLEVIEWER_H
+#define GLTFIO_SIMPLEVIEWER_H
+
+#include <filament/Box.h>
+#include <filament/IndirectLight.h>
+#include <filament/LightManager.h>
+#include <filament/Material.h>
+#include <filament/RenderableManager.h>
+#include <filament/Scene.h>
+#include <filament/TransformManager.h>
+#include <filament/View.h>
+
+#include <gltfio/Animator.h>
+#include <gltfio/AssetLoader.h>
+#include <gltfio/FilamentAsset.h>
+
+#include <math/mat4.h>
+#include <math/vec3.h>
+
+#include <utils/Entity.h>
+#include <utils/NameComponentManager.h>
+
+#include <imgui.h>
+
+#include <vector>
+
+namespace gltfio {
+
+/**
+ * SimpleViewer manages the state for a simple glTF viewer with imgui controls and a tree view.
```

Comment:
Are the ImGui controls optional? It could be useful to be able to just shown the scene.

Reply:
If you don't want ImGui controls, then there is no need to use this class.  I updated the class comment to clarify.